internal/dbutil/pgutil/db.go

```diff
@@ -98,3 +100,17 @@ func QuerySnapshot(db dbschema.Queryer) (*dbschema.Snapshot, error) {
 		Data:    data,
 	}, err
 }
+
+//CheckApplicationName ensures that the Connection String contains an application name
+func CheckApplicationName(s string) (r string) {
+	if !strings.Contains(s, "ApplicationName") {
```

Comment:
shouldn't you be checking for `application_name` instead of `ApplicationName`?

Reply:
Done, good catch!